### PR TITLE
Test config update

### PR DESCRIPTION
**Pull request by leofigy**
## Description

Hello @themantissa , @coderGo93  Updating cluster test config, because it fails because the diff 
Link to any related issue(s):

## Type of change:

- [X] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [X] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [X] I have read the Terraform contribution guidelines
- [X] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [X] I have added any necessary documentation (if appropriate)
- [X] I have run make fmt and formatted my code

### CODE REVIEW

File: mongodbatlas/data_source_mongodbatlas_cloud_provider_snapshot_backup_policy_test.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func testAccMongoDBAtlasDataSourceCloudProviderSnapshotBackupPolicyConfig(projec
 			provider_region_name        = "EU_CENTRAL_1"
 			provider_instance_size_name = "M10"
 			provider_backup_enabled     = true //enable cloud provider snapshots
-			provider_disk_iops          = 100
+			provider_disk_iops          = 3000
```
Comment (themantissa) [inline mongodbatlas/data_source_mongodbatlas_cloud_provider_snapshot_backup_policy_test.go]: This isn't required.   We recently removed it from the examples due to the issues with it and that it is not required, especially for M10s (where you can't set a higher iops than you get).   I'd just remove it.

**Review by themantissa — CHANGES_REQUESTED** [review]
Thank you for doing this!  Since it's not required I'd just remove it vs updating.  Long term likely to be more sustainable.

File: mongodbatlas/data_source_mongodbatlas_cloud_provider_snapshot_backup_policy_test.go (hunk 1 above)
Comment (leofigy) [reply mongodbatlas/data_source_mongodbatlas_cloud_provider_snapshot_backup_policy_test.go]: okay that's even better :) removing it

**Review by themantissa — APPROVED** [review]
LGTM!  Thanks :)

**Pull request merged**